tests/scripts/thread-cert/mesh_cop.py

```diff
@@ -0,0 +1,832 @@
+        major = rest[3] & 0xf
+        return VendorStackVersion(stack_vendor_oui, build, rev, minor, major)
+
+
+# UdpEncapsulation TLV (48)
+class UdpEncapsulation(object):
+    # TODO: Not implemented yet
+    pass
+
+
+class UdpEncapsulationFactory(object):
+    # TODO: Not implemented yet
+
+    def parse(self, data, message_info):
+        raise NotImplementedError("TODO: Not implemented yet")
+
+
+# Ipv6Address TLV (49)
+class Ipv6Address(object):
+    # TODO: Not implemented yet
+    pass
+
+
+class Ipv6AddressFactory(object):
+    # TODO: Not implemented yet
+
+    def parse(self, data, message_info):
+        raise NotImplementedError("TODO: Not implemented yet")
+
+
+# PendingTimestamp TLV (51)
+class PendingTimestamp(object):
+    # TODO: Not implemented yet
+    pass
+
+
+class PendingTimestampFactory(object):
+    # TODO: Not implemented yet
+
+    def parse(self, data, message_info):
+        raise NotImplementedError("TODO: Not implemented yet")
+
+
+# DelayTimer TLV (52)
+class DelayTimer(object):
+    # TODO: Not implemented yet
+    pass
+
+
+class DelayTimerFactory(object):
+    # TODO: Not implemented yet
+
+    def parse(self, data, message_info):
+        raise NotImplementedError("TODO: Not implemented yet")
+
+
+# ChannelMask TLV (53)
+class ChannelMask(object):
+    # TODO: Not implemented yet
+    pass
+
+
+class ChannelMaskFactory(object):
+    # TODO: Not implemented yet
+
+    def parse(self, data, message_info):
+        raise NotImplementedError("TODO: Not implemented yet")
+
+
+# Count TLV (54)
+class Count(object):
+    # TODO: Not implemented yet
+    pass
+
+
+class CountFactory(object):
+    # TODO: Not implemented yet
+
+    def parse(self, data, message_info):
+        raise NotImplementedError("TODO: Not implemented yet")
+
+
+# Period TLV (55)
+class Period(object):
+    # TODO: Not implemented yet
+    pass
+
+
+class PeriodFactory(object):
+    # TODO: Not implemented yet
+
+    def parse(self, data, message_info):
+        raise NotImplementedError("TODO: Not implemented yet")
+
+
+# ScanDuration TLV (56)
+class ScanDuration(object):
+    # TODO: Not implemented yet
+    pass
+
+
+class ScanDurationFactory(object):
+    # TODO: Not implemented yet
+
+    def parse(self, data, message_info):
+        raise NotImplementedError("TODO: Not implemented yet")
+
+
+# EnergyList TLV (57)
+class EnergyList(object):
+    # TODO: Not implemented yet
+    pass
+
+
+class EnergyListFactory(object):
+    # TODO: Not implemented yet
+
+    def parse(self, data, message_info):
+        raise NotImplementedError("TODO: Not implemented yet")
+
+
+
+class MeshCopCommand(object):
+
+    def __init__(self, _type, tlvs):
+        self._type = _type
+        self._tlvs = tlvs
+
+    @property
+    def type(self):
+        return self._type
+
+    @property
+    def tlvs(self):
+        return self._tlvs
+
+    def __repr__(self):
+        tlvs_str = ", ".join(["{}".format(tlv) for tlv in self.tlvs])
```

Comment:
I the `__repr__` of every tlvs will be automatically called if you directly pass a list of tlv to `format()` funtion.

Reply:
It seems that the default function will print the memory address of that object, and the existing tlvs     also defined __repr__ function. I think it would be better define the behavior clearly.